src/Features/Core/Portable/EditAndContinue/RudeEditKind.cs

```diff
@@ -97,7 +97,11 @@ internal enum RudeEditKind : ushort
         ReadOnlyStruct = 78,
         ReadOnlyReferences = 79,
 
+		// MERGE TODO
+		// -- InternalError added in 15.9 via https://github.com/dotnet/roslyn/commit/6f6910c4d0a4fc60ccfd79efd8d54b2f45596aa1#diff-deefc0664e50cdcdda9fc7afcdd93887
+		// -- InsertMethodWithExplicitInterfaceSpecifier added in master via https://github.com/dotnet/roslyn/commit/0f8d61c2b21422871ebe87e895176849f9a3e119#diff-deefc0664e50cdcdda9fc7afcdd93887
         InsertMethodWithExplicitInterfaceSpecifier = 80,
+        InternalError = 80,
```

Comment:
@tmat @ivanbasov Please advise on what to do here. I'm guessing we change `InsertMethodWithExplicitInterfaceSpecifier` to 81 since it's only in master and therefore shouldn't have (external) telemetry yet...